extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/OidcBuildStep.java

```diff
@@ -46,7 +46,7 @@ EnableAllSecurityServicesBuildItem security() {
 
     @Record(ExecutionTime.RUNTIME_INIT)
     @BuildStep
-    public void setup(OidcConfig config, OidcRecorder recorder, VertxBuildItem vertxBuildItem,
+    public void setup(OidcConfig config, OidcRecorder recorder, CoreVertxBuildItem vertxBuildItem,
```

Comment:
Actually this should be using io.quarkus.vertx.core.deployment.InternalWebVertxBuildItem for now. We want OIDC to be on the same vert.x instance as HTTP.

Reply:
@stuartwdouglas It is deprecated so I used a qualified parameter name to avoid suppressing the deprecation warnings everywhere :-)

Reply:
It's deprecated because the intention was to remove it for 1.0, but it did not make it in. This is the correct one to use here though. We want to get rid of it, but we just did not have time.

Reply:
OK